wgengine/filter/tailcfg.go

```diff
@@ -18,6 +18,7 @@ var defaultProtos = []ipproto.Proto{
 	ipproto.UDP,
 	ipproto.ICMPv4,
 	ipproto.ICMPv6,
+	ipproto.GRE,
```

Comment:
we don't want to add GRE to the defaultProtos. It needs to be opt-in for users, like we did for SCTP.

Reply:
@bradfitz 
Thank you for your comment.
To make GRE as opt-in option is nice idea.

I checked about SCTP and it seems to be allowed by port based ACL.
The problem is GRE doesn't have any port - like ICMP - and current ACL doesn't have protocol based rule.
Do you have any idea ?